roles/ovirt/tasks/retrieve_vm_ip.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Set vm name as variable
```

Comment:
No need to use this fact and run a task only for this. You can move this fact as a plain variable [here](https://github.com/openstack-k8s-operators/ci-framework/pull/1132/files#diff-a054414bb42bbceda6a238db6877eec76f12a4e56dd3b4c9efc220c1205ae376R58).

Reply:
Thanks, Fixed.